tests/conftest.py

```
@@ -0,0 +1,274 @@
+
+    This ensures each test starts with a fresh vector database state.
+    """
+    try:
+        async with VectorService() as vector_service:
+            # Delete and recreate collection for clean state
+            try:
+                await vector_service._client.delete_collection(
+                    test_settings.qdrant_collection
+                )
+            except Exception:
+                pass  # Collection might not exist
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)